src/Microsoft.Health.Fhir.Core/Features/Operations/Export/ExportJobTask.cs

```diff
@@ -198,6 +198,13 @@ public async Task ExecuteAsync(ExportJobRecord exportJobRecord, WeakETag weakETa
                 _exportJobRecord.FailureDetails = new JobFailureDetails(ex.Message, HttpStatusCode.BadRequest);
                 await CompleteJobAsync(OperationStatus.Failed, cancellationToken);
             }
+            catch (FailedToAnonymizeResourceException ex)
+            {
+                _logger.LogError(ex, "Failed to anonymize resource. The job will be marked as failed.");
+
+                _exportJobRecord.FailureDetails = new JobFailureDetails(ex.Message, HttpStatusCode.BadRequest);
```

Comment:
>ex.Message [](start = 72, length = 10)

nit: this message gets returned to the customer as part of OperationOutcome Issue. Would it be better to have a standardized failure message here instead of the exception message since we are catching a generic `Exception` below?

Reply:
Sure, add more detail message.